.github/workflows/on_release_branch.yml

```diff
@@ -0,0 +1,171 @@
+name: On release branch
+
+on:
```

Comment:
This workflow is for master, release, and release tags (e.g. v1.0.0). 
This would assemble a release build, upload to deploygate, run lints and/or tests for now.